packages/lib/src/vis/shared/SelectionLine.tsx

```diff
@@ -31,23 +17,32 @@ function SelectionLine(props: Props) {
     color = 'black',
   } = props;
 
-  const { dataToWorld } = useAxisSystemContext();
-  const startPoint = dataToWorld(dataStartPoint);
-  const endPoint = dataToWorld(dataEndPoint);
+  const { width, height } = useThree((state) => state.size);
 
-  const [dataGeometry] = useState(() => new BufferGeometry());
-  const invalidate = useThree((state) => state.invalidate);
+  const { dataToWorld } = useAxisSystemContext();
+  const worldToHtml = useWorldToHtml();
 
-  useLayoutEffect(() => {
-    dataGeometry.setFromPoints([startPoint, endPoint]);
-    dataGeometry.computeBoundingSphere();
-    invalidate();
-  }, [dataGeometry, endPoint, invalidate, startPoint]);
+  const worldStartPoint = dataToWorld(dataStartPoint);
+  const htmlStartPt = worldToHtml(
+    new Vector3(worldStartPoint.x, worldStartPoint.y, 0)
+  );
+  const worldEndPoint = dataToWorld(dataEndPoint);
+  const htmlEndPt = worldToHtml(
+    new Vector3(worldEndPoint.x, worldEndPoint.y, 0)
+  );
 
   return (
-    <line_ geometry={dataGeometry}>
-      <lineBasicMaterial color={color} />
-    </line_>
+    <Html>
+      <svg width={width} height={height}>
+        <line
+          x1={htmlStartPt.x}
+          y1={htmlStartPt.y}
+          x2={htmlEndPt.x}
+          y2={htmlEndPt.y}
+          stroke={color}
```

Comment:
Maybe the props from the component should refer to the svg now (e.g. set `stroke` instead of `color` for the prop name).

Reply:
Hmm, this would indeed be consistent with visx components.

Reply:
We could even have `Props` extend the svg props. That may be a step too far but would facilitate the use of a dashed line for #999 for example.

Reply:
Yep, I'm down with that.